src/devices/ctm.ts

```diff
@@ -637,6 +648,48 @@ const definitions: Definition[] = [
         fromZigbee: [fz.on_off, fz.brightness, fz.lighting_ballast_configuration],
         toZigbee: [tz.on_off, tz.light_onoff_brightness, tz.light_brightness_move, tz.ballast_config],
         meta: {disableDefaultResponse: true},
+        ota: ota.zigbeeOTA,
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl', 'lightingBallastCfg']);
+            await endpoint.read('genOnOff', ['onOff']);
+            await reporting.onOff(endpoint);
+            await endpoint.read('genLevelCtrl', ['currentLevel']);
+            await reporting.brightness(endpoint);
+            await endpoint.read('lightingBallastCfg', ['minLevel', 'maxLevel', 'powerOnLevel']);
+            await endpoint.configureReporting('lightingBallastCfg', [{
+                attribute: 'minLevel',
+                minimumReportInterval: 0,
+                maximumReportInterval: constants.repInterval.HOUR,
+                reportableChange: null}]);
+            await endpoint.configureReporting('lightingBallastCfg', [{
+                attribute: 'maxLevel',
+                minimumReportInterval: 0,
+                maximumReportInterval: constants.repInterval.HOUR,
+                reportableChange: null}]);
+            await endpoint.configureReporting('lightingBallastCfg', [{
+                attribute: 'powerOnLevel',
+                minimumReportInterval: 0,
+                maximumReportInterval: constants.repInterval.HOUR,
+                reportableChange: null}]);
+        },
+        exposes: [e.light_brightness(),
+            e.numeric('ballast_minimum_level', ea.ALL).withValueMin(1).withValueMax(99)
+                .withDescription('Specifies the minimum brightness value'),
+            e.numeric('ballast_maximum_level', ea.ALL).withValueMin(1).withValueMax(99)
+                .withDescription('Specifies the maximum brightness value'),
+            e.numeric('ballast_power_on_level', ea.ALL).withValueMin(1).withValueMax(99)
+                .withDescription('Specifies the initialisation light level. Can not be set lower than "ballast_minimum_level"')],
+    },
+    {
+        zigbeeModel: ['DimmerPille'],
+        model: 'CTM_DimmerPille',
```

Comment:
Looks the code of this device exactly the same as `mTouch_Dim`, can you add it as a whitelabel instead? Example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/c97e3e51d6801bd6bd265f617e14b1d944001dd4/src/devices/bosch.ts#L1369